IPython/utils/tests/test_text.py

```diff
@@ -80,13 +80,13 @@ def eval_formatter_check(f):
     nt.assert_equal(s, ns['stuff'])
     s = f.format("{stuff!r}", **ns)
     nt.assert_equal(s, repr(ns['stuff']))
-    
+
```

Comment:
Looks like you have your editor configured to mess about with whitespace. We would prefer that you avoid committing large whitespace changes.

If you are using atom, this is its extremely unfortunate default behavior, and can be disabled with: Preferences > Packages > whitespace > disable